meetings/2022-03-14.md

```diff
@@ -0,0 +1,108 @@
+* eP: We discussed having a public key that could proove the token was issued by someone trusted.
+* AC: We would have to be careful about token rotation.
+* eP: Yes, with per user key it's probably less frequent need, but it would indeed put some management burden on the user since they would have to do it.
+* MB: If this scenario is one that we would support, yes the user would need to manage key rotation. 
+* eP: If you want to avoid rotating that, it's a kind of trade-off. There could be a standard period for the user to update their profile. Some services prompt a user to change their password regularly, it seems reasonable.
+* MB: Issuer restrictions would get more complex here. If issuer is in charge of managing key pairs it could be a piece of software on your machine.
+* eP: Do you refer to self-issued OIDC like flows?
+* MB: Yes, it is de-facto the same kind of thing (or a possible implementation thereof).
+* eP: How to follow up on this issue/conversation? Is anyone interested in proposing initial text of how to address it?
+* MB: It might be interesting to involve people behind WebID-TLS. The main problem was usability issue, which was never resolved on the browser level. If that could be resolved server side.
+* eP: Would you like to reach out Matthieu? I'll try to reach out to the self-sovereign identity people.
+* MB: Yes, I'll reach out.
+* AC: In terms of scope I'd like to make sure that we make it clear what is in scope for Solid-OIDC. I think self-issued OIDC is much more in scope. When it comes to WebID-TLS I would see it outside of the scope. I would rather see multiple documents rather than one complicated with forking options.
+* MB: At this stage, I only mentioned WebID-TLS just because it relies on public/private key pair. I would ask what about managing private key on the server side. Maybe the two aren't incompatible.
+* AC: One thing to keep in mind. When it comes to existing OIDC infrastructure and ecosystem. The more requirements we put on OP, the harder it is for existing ecosystem to adjust. 
+* ...:  WebID scope (fairly typical, adding new scopes to OPs)
+* ...: DPoP which is draft and moving through to the OAuth working group
+* ...: Entirely Solid-OIDC is the client identifiers which is a little bit of a challenge for new providers to support...
+* Anything we add akin to that 3rd category/the more requirements we add to OPs, the harder it will be for existing ones in the ecosystem to come along with where we're going.
+* EP: I'm thinking the core spec could have a generic working as an etension, but not as a requirement (issuer delegation/publishing public key). I think it would be a completely different spec and let some server implement that. In my case, if there's no IdP restriction, that's fine, if there are restrictions, then the user is stuck with that OP if they want to use data.
```

Comment:
```suggestion
* EP: I'm thinking the core spec could have a generic wording allowing it as an etension, but not as a requirement (issuer delegation/publishing public key). I think it would be a completely different spec and let some server implement that. In my case, if there's no IdP restriction, that's fine, if there are restrictions, then the user is stuck with that OP if they want to use data.
```